modules/bootkube/resources/bootkube.sh

```diff
@@ -29,14 +29,13 @@ cp "$PWD/tnc-bootstrap/tectonic-node-controller-config.yaml" /etc/kubernetes/tnc
 # We originally wanted to run the etcd cert signer as
 # a static pod, but kubelet could't remove static pod
 # when API server is not up, so we have to run this as
-# docker container.
+# podman container.
 # See https://github.com/kubernetes/kubernetes/issues/43292
 
 echo "Starting etcd certificate signer..."
 
 # shellcheck disable=SC2154
-SIGNER=$(/usr/bin/docker run -d \
-    --tmpfs /tmp \
```

Comment:
i think this was basically for security so that the keys/cert never hit the disk